common/hooks/do-pkg/00-gen-pkg.sh

```diff
@@ -23,7 +23,14 @@ genpkg() {
 		return 0
 	fi
 
+	# Lock binpkg
+	unlock_binpkg() {
+	    msg_normal "Unlocking ${binpkg}.lock because an error occurred during $binpkg creation"
+	    rm -f $pkgdir/${binpkg}.lock
+	}
+
 	touch -f $pkgdir/${binpkg}.lock
+	trap unlock_binpkg ERR
```

Comment:
Does trapping `ERR` cover enough? Or do we also need `EXIT`?

Reply:
Yep need `EXIT` error or else a C-c wouldn't be caught. I tested by adding a 100 second sleep and hitting C-c, of course it didn't trap so the lock still dangled.